extension/src/pipeline/quickPick.ts

```diff
@@ -92,7 +92,7 @@ const pickFields = async (
     { title: Title.SELECT_PLOT_Y_METRIC }
   )) as { file: string; key: string }[] | undefined
 
-  if (!yValues) {
+  if (!yValues || yValues.length === 0) {
```

Comment:
We could also add a toast message if the user selects 0 y metrics if preferred but I thought it would be obvious enough :)